### PR TITLE
Document Azure Blob Storage source

### LINKED ISSUE #110: Create docs for Azure Storage source
Currently there are none

### PR DESCRIPTION
**Pull request by antoineco**
[Render](https://github.com/antoineco/triggermesh-docs/blob/azure-blob-storage/docs/sources/azureblobstorage.md)

Closes #110

### CODE REVIEW

File: docs/sources/azureblobstorage.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+![Event Hub overview](../images/azureblobstorage-source/eventhub-1.png)
+![Event Hub resource ID](../images/azureblobstorage-source/eventhub-2.png)
+
+Resource IDs can also be obtained using the [Azure CLI][azure-cli] (`az`). The following command line uses values from
+the screenshots above:
+
+```console
+$ az eventhubs eventhub show --resource-group blobstorage-source-dev --namespace-name eventsourcedev --name my-event-hub
+TODO
```
Comment (sebgoa) [inline docs/sources/azureblobstorage.md:104]: why is there a `TODO` here ?

File: docs/sources/azureblobstorage.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+```
+
+## Deploying an Instance of the Source
+
+Open the Bridge creation screen and add a source of type `Azure Blob Storage`.
+
+![Adding an Azure Blob Storage source](../images/azureblobstorage-source/create-bridge-1.png)
```
Comment (sebgoa) [inline docs/sources/azureblobstorage.md:111]: the following three images are not rendered

Comment (sebgoa) [reply docs/sources/azureblobstorage.md:111]: they are missing in the PR

**Review by sebgoa — CHANGES_REQUESTED** [review]
missing images it seems

File: docs/sources/azureblobstorage.md (hunk 1 above)
Comment (antoineco) [reply docs/sources/azureblobstorage.md:104]: Please wait, the PR is a draft. We don't have a form yet so there will be things missing.

**Comment by sebgoa** [comment]
went through the rendered page

+1

**Review by sebgoa — APPROVED** [review]

**Pull request merged**